tests/cypress/support/commands.js

```diff
@@ -30,12 +30,18 @@ Cypress.Commands.add('createAnnotationTask', (taksName='New annotation task',
                                               attrName='Some attr name',
                                               textDefaultValue='Some default value for type Text',
                                               image='image.png',
-                                              multiJobs=false,
-                                              segmentSize=1,
                                               multiAttr=false,
                                               additionalAttrName,
                                               typeAttribute,
-                                              additionalValue) => {
+                                              additionalValue,
+                                              advancedConfiguration=false,
+                                              multiJobs,
+                                              segmentSize,
+                                              sssFrame,
+                                              startFrame,
+                                              stopFrame,
+                                              frameStep
+                                              ) => {
```

Comment:
I would suggest to pass object here to this function uses available fields from the object

Reply:
Thanks. Replaced. 
In 2 tests, I also had to slightly correct the task creation function.